### PR TITLE
Client: introduce a low-level client for the v2 API

### LINKED ISSUE #42: Feature request dbaas
Hello

I wanted to ask if it is planned to extend the API with DBAAS so one is able to spawn and destroy databases ?

### LINKED ISSUE #43: Doc Website CSS Broken
Dear Exoscale Team, just to let you know that the CSS of your documentation website is broken 

https://exoscale.github.io/python-exoscale/

It looks like that to me

image

### PR DESCRIPTION
**Pull request by brutasse**
This client is dynamically generated from the OpenAPI definition. It expands each API call into a method on the client class.

A scheduled action is added to poll the OpenAPI definition for updating the in-repo copy.

Fixes #43
Fixes #42

### CODE REVIEW

**Comment by brutasse** [comment]
To hint at what this results in:

ksnip_20230511-100411

File: .sort-enums.py
Comment (exo-cedric) [inline .sort-enums.py:1]: Do we have to make this a _hidden_ file ?

**Review by exo-cedric — APPROVED** [review]
(does the auto-generated doc include a small usage example ?)

**Review by zyegfryed — APPROVED** [review]
Great job, lgtm 👍

File: .sort-enums.py
Comment (brutasse) [reply .sort-enums.py:1]: It's not strictly necessary, just to minimize the amount of visible boilerplate files when checking out the repo.

**Comment by brutasse** [comment]
> (does the auto-generated doc include a small usage example ?)

Nothing beyond what can be extracted from the OpenAPI specification yet. Ideally this would happen by further enriching the spec and the parsing of it.

**Pull request merged**